docs/en/rules/Azure.Resource.adminUsername.md

```diff
@@ -0,0 +1,22 @@
+---
+severity: Awareness
+pillar: Security
+category: Design
+resource: All resources
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Resource.adminUsername/
+---
+
+# Administrator Username Types
+
+## SYNOPSIS
+
+The adminUsername or administratorUsername property should not be a string literal in nested templates.
+
+## DESCRIPTION
```

Comment:
We need to add any links. Added some.

Reply:
@BernieWhite I've add 2 links in. They're about how to use Azure KeyVault in Bicep / ARM templates. Anything else you think I should include?